gbus/abstractions.go

```diff
@@ -149,6 +158,10 @@ type Builder interface {
 
 	//WithPolicies defines the default policies that are applied for evey outgoing amqp messge
 	WithPolicies(policies ...MessagePolicy) Builder
+
+	//WithDbPingTimeout defines the default timeout in seconds for the db ping check
+	WithDbPingTimeout(timeoutInSeconds time.Duration) Builder
```

Comment:
Let's change this to ConfigureHealthCheck as I wouldn't want to add additional methods if we want to evolve the ability to configure the health check later on the road.